pylib/src/lib.rs

```diff
@@ -6,9 +6,9 @@
 // so build.rs sets this env var
 pub const LIB_PATH: &str = match option_env!("win_lib_path") {
     Some(s) => s,
-    None => concat!(env!("CARGO_MANIFEST_DIR"), "/Lib"),
+    None => concat!(env!("CARGO_MANIFEST_DIR"), "../Lib"),
```

Comment:
during building `rustpython-pylib`, `Lib/` is copied to under `pylib/Lib/`. So this change is not necessary.
somehow in your environemnt, `Lib/` looks not copied under `pylib`. Could you check why it didn't?

Reply:
It's not copied, it appears to be a symlink, which itself interpreted as a plain text file in windows if it is a unix symbolic link.

Reply:
I don't remember well about windows build, but because our CI is running also on windows environment, processing that part looks having bugs in some cases.
Could you look more in there why it fails on your machine?

Reply:
OKay, will let you know

Reply:
Hey, I able to compile without errors when using `RUSTPYTHONPATH` to correct directory